winui/DataGrid/Master-Details-View.md

```diff
@@ -299,106 +306,113 @@ To define Master-Details View relations, create [GridViewDefinition](https://hel
+                </dataGrid:SfDataGrid>
+            </dataGrid:GridViewDefinition.DataGrid>
+        </dataGrid:GridViewDefinition>
+    </dataGrid:SfDataGrid.DetailsViewDefinition>
+</dataGrid:SfDataGrid>
 {% endhighlight %}
 {% highlight c# %}
 dataGrid.AutoGenerateRelations = false;
 
 // GridViewDefinition for parent DataGrid
 var gridViewDefinition1 = new GridViewDefinition();
-gridViewDefinition1.RelationalColumn = "OrderDetails";
-var firstLevelNestedGrid = new SfDataGrid() { Name = "FirstLevelNestedGrid", AutoGenerateColumns = true };
+gridViewDefinition1.RelationalColumn = "Sales";
+var firstLevelNestedGrid = new SfDataGrid() { Name = "FirstLevelNestedGrid", AutoGenerateColumns = true, GridLinesVisibility = Syncfusion.UI.Xaml.Grids.GridLinesVisibility.Both };
 firstLevelNestedGrid.AutoGenerateRelations = false;
 
 // GridViewDefinition for FirstLevelNestedGrid
 var gridViewDefinition = new GridViewDefinition();
-gridViewDefinition.RelationalColumn = "SalesDetails";
-gridViewDefinition.DataGrid = new SfDataGrid() { Name = "SecondLevelNestedGrid", AutoGenerateColumns = true };
+gridViewDefinition.RelationalColumn = "Orders";
+gridViewDefinition.DataGrid = new SfDataGrid() { Name = "SecondLevelNestedGrid", AutoGenerateColumns = true, GridLinesVisibility = Syncfusion.UI.Xaml.Grids.GridLinesVisibility.Both };
 firstLevelNestedGrid.DetailsViewDefinition.Add(gridViewDefinition);
 gridViewDefinition1.DataGrid = firstLevelNestedGrid;
 
 dataGrid.DetailsViewDefinition.Add(gridViewDefinition1);
 {% endhighlight %}
 {% endtabs %}
 
-![WinUI DataGrid displays Master Details View with Manually defined Relations](Master-Details-View-Images/winui-datagrid-master-details-view-manual-defined-relation.png)
+<img src="Master-Details-View-Images/winui-datagrid-master-details-view-manual-defined-relation.png" alt="WinUI DataGrid displays Master Details View with Manually defined Relations" width="100%" Height="Auto"/>
 
 ## Populating Master-Details view through events
 
 You can load `ItemsSource` for [DetailsViewDataGrid](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.DetailsViewDataGrid.html) asynchronously by handling [SfDataGrid.DetailsViewExpanding](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.SfDataGrid.html#Syncfusion_UI_Xaml_DataGrid_SfDataGrid_DetailsViewExpanding). You can set ItemsSource in on-demand when expanding record through [GridDetailsViewExpandingEventArgs.DetailsViewItemsSource](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.GridDetailsViewExpandingEventArgs.html#Syncfusion_UI_Xaml_DataGrid_GridDetailsViewExpandingEventArgs_DetailsViewItemsSource) property in the [SfDataGrid.DetailsViewExpanding](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.SfDataGrid.html#Syncfusion_UI_Xaml_DataGrid_SfDataGrid_DetailsViewExpanding) event handler. 
 
 {% tabs %}
 {% highlight xaml %}
-<syncfusion:SfDataGrid  x:Name="dataGrid"
+<dataGrid:SfDataGrid  x:Name="dataGrid"
```

Comment:
Some of the MasterDetailsView images Header text should be change with space.

Reply:
Modified the images